# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../metamail/10_mm-2.7-glibc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mm2.7/src/metamail/metamail.c.ewt	Mon Aug 26 17:16:59 1996
+++ mm2.7/src/metamail/metamail.c	Mon Aug 26 17:17:12 1996
@@ -48,6 +48,7 @@
 #ifndef AMIGA
 #ifdef SYSV
 #include <termio.h>
+#include <sys/ioctl.h>
 #include <unistd.h>
 #else /* SYSV */
 #include <sgtty.h>
